cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -172,6 +173,7 @@ private boolean isKafkaAssignerMode(Collection<String> goals) {
    *                                  (if null, use num.concurrent.leader.movements).
    * @param skipHardGoalCheck True if the provided {@code goals} do not have to contain all hard goals, false otherwise.
    * @param excludedTopics Topics excluded from partition movement (if null, use topics.excluded.from.partition.movement)
+   * @param replicaMovementStrategy The strategy used to determine the execution order of generated replica movement tasks.
```

Comment:
(This applies to all relevant JavaDoc for `replicaMovementStrategy`) Can we specify the expected behavior when the value is `null`? -- please see JavaDoc for `excludedTopics` for example.